helm-chart/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: hedera-json-rpc-relay
+description: Helm chart deployment of the hashgraph/hedera-json-rpc-relay
+type: application
+version: 0.0.1
+appVersion: "0.0.1"
+home: https://github.com/hashgraph/hedera-json-rpc-relay
```

Comment:
nit: add an extra line to all new files to git picks up commit changes more easily.
I also forgot this on one of my more recent PRs so i get it.